go.mod

```diff
@@ -3,15 +3,13 @@ module github.com/dymensionxyz/e2e-tests
 go 1.22.1
 
 replace (
-	github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
-	github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
 	github.com/evmos/ethermint => github.com/dymensionxyz/ethermint v0.22.0-dymension-v0.3
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/strangelove-ventures/interchaintest/v6 => github.com/faddat/ibctest/v6 v6.0.0-20240402123907-c59466726138
```

Comment:
Why we need to use interchaintest from strangelove?

Reply:
The repo itself has already used [decentrio framework](https://github.com/decentrio/rollup-e2e-testing) to make the test compatible with dymension chain and rollapps.